recipes/libtiff/all/conanfile.py

```diff
@@ -185,24 +183,25 @@ def package(self):
         self.copy("COPYRIGHT", src=self._source_subfolder, dst="licenses", ignore_case=True, keep_path=False)
         cmake = self._configure_cmake()
         cmake.install()
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
         self.cpp_info.set_property("cmake_find_mode", "both")
         self.cpp_info.set_property("cmake_file_name", "TIFF")
         self.cpp_info.set_property("cmake_target_name", "TIFF::TIFF")
-        self.cpp_info.set_property("pkg_config_name", "libtiff-{}".format(tools.Version(self.version).major))
+        self.cpp_info.set_property("pkg_config_name", "libtiff-{}".format(Version(self.version)[0]))
         if self.options.cxx:
             self.cpp_info.libs.append("tiffxx")
         self.cpp_info.libs.append("tiff")
-        if self.settings.os == "Windows" and self.settings.build_type == "Debug" and self._is_msvc:
+        if self.settings.os == "Windows" and self.settings.build_type == "Debug" and is_msvc(self):
             self.cpp_info.libs = [lib + "d" for lib in self.cpp_info.libs]
-        if self.options.shared and self.settings.os == "Windows" and not self._is_msvc:
+        if self.options.shared and self.settings.os == "Windows" and not is_msvc(self):
             self.cpp_info.libs = [lib + ".dll" for lib in self.cpp_info.libs]
         if self.settings.os in ["Linux", "Android", "FreeBSD", "SunOS", "AIX"]:
             self.cpp_info.system_libs.append("m")
 
         # TODO: to remove in conan v2 once cmake_find_package* & pkg_config generators removed
         self.cpp_info.names["cmake_find_package"] = "TIFF"
         self.cpp_info.names["cmake_find_package_multi"] = "TIFF"
-        self.cpp_info.names["pkg_config"] = "libtiff-{}".format(tools.Version(self.version).major)
+        self.cpp_info.names["pkg_config"] = "libtiff-{}".format(Version(self.version)[0])
```

Comment:
To fix once https://github.com/conan-io/conan/pull/11823 released in a version we can use in cci?

Reply:
As soon as we have 1.52.0 or 1.51.2 running on CCI.